driver-async/src/test/functional/com/mongodb/async/client/gridfs/GridFSBucketSmokeTestSpecification.groovy

```diff
@@ -119,6 +120,95 @@ class GridFSBucketSmokeTestSpecification extends FunctionalSpecification {
         'a large file to stream' | true       | 5          | false
     }
 
+    def 'should round trip with small chunks'() {
```

Comment:
I'd like to see a test that exercises the case where the file size is larger than DEFAULT_BUFFER_SIZE.  Maybe just increase the size of contentBytes in this one?